app/presenters/report_presenter.rb

```diff
@@ -13,6 +13,6 @@ def deadline
 
   def financial_quarter_and_year
     return nil if financial_quarter.nil? || financial_year.nil?
-    "Q#{financial_quarter} #{financial_year}-#{financial_year + 1}"
+    "FQ#{financial_quarter} #{financial_year}-#{financial_year + 1}"
```

Comment:
My branch `feature/1439-add-historic-actuals-transactions-to-the-export-csv` removes this method and all other implementations of `financial_quarter_and_year` into a single module -- does this method need to be different for reports, or should all instances of `financial_quarter_and_year` have an `F` at the start?

Reply:
I don't know, this is one of those cases where I don't want to build a bikeshed for us to paint, but I need to find out if this "Q" versus "FQ" is really important for the users. (It seems *to me* that having it consistent everywhere would be preferable, but there might be business logic reasons I'm missing.)